coral/src/domain/environment/environment-api.ts

```diff
@@ -9,4 +12,17 @@ const getEnvironments = async (): Promise<Environment[]> => {
     .then(transformEnvironmentApiResponse);
 };
 
-export { getEnvironments };
+const getClusterInfo = async ({
+  envSelected,
+  envType,
+}: {
+  envSelected: string;
+  envType: string;
+}): Promise<ClusterInfo> => {
+  const params = new URLSearchParams({ envSelected, envType });
```

Comment:
`new URLSearchParams` is not constructed twice

Reply:
Oops, thank you!